hbase-zookeeper/src/main/java/org/apache/hadoop/hbase/zookeeper/ZKUtil.java

```diff
@@ -1400,9 +1399,16 @@ public static void deleteNodeRecursivelyMultiOrSequential(ZKWatcher zkw,
    * Chunks the provided {@code ops} when their approximate size exceeds the the configured limit.
    * Take caution that this can ONLY be used for operations where atomicity is not important,
    * e.g. deletions. It must not be used when atomicity of the operations is critical.
+   *
+   * @param zkw                         ZKWatcher
```

Comment:
Please make it just a single whitespace between the parameter and the description. Also, please try to make it more descriptive.